### ISSUE TITLE
Non-owner Origin members cannot search for or install private packages

### ISSUE THREAD

**Issue by bixu** [habitat-sh/builder#1224]
### `hab` CLI version
```% hab --version
hab 0.88.0/20191009205151```

### Platform Details
```% uname -a
Darwin Halation-Effect.local 19.0.0 Darwin Kernel Version 19.0.0: Thu Oct 17 16:17:15 PDT 2019; root:xnu-6153.41.3~29/RELEASE_X86_64 x86_64```
Also on AWS instances running Ubuntu 16.04

### Scenario:
We have created GitHub 'service users' that are members of our private Origin, but when they search for or try to install private packages from this private Origin they cannot see or install these packages.

### Steps to Reproduce:
```% hab pkg search mybuildings/api --auth=<service user's auth token>
No packages found that match 'mybuildings/api'```

### Expected Result:
We expect to see the same search/install results as the Origin owner does (an engineer who work on our team).

### Actual Result:
```% RUST_LOG=debug hab pkg search mybuildings/api --auth=XXXXX
[2019-11-12T10:45:41Z DEBUG habitat_common::ui] UI { shell: Shell { input: InputStream { isatty: false }, out: OutputStream { coloring: Never, isatty: false }, err: OutputStream { coloring: Never, isatty: false } } }
[2019-11-12T10:45:41Z DEBUG hab] clap cli args: ["hab", "pkg", "search", "mybuildings/api", "--auth=XXXXX"]
[2019-11-12T10:45:41Z DEBUG hab] remaining cli args: []
[2019-11-12T10:45:41Z DEBUG hab::config] Loading CLI config from /Users/irvin/.hab/etc/cli.toml
[2019-11-12T10:45:41Z DEBUG habitat_http_client::api_client] Client socket timeout: 300 secs
[2019-11-12T10:45:41Z DEBUG habitat_http_client::api_client] Skip cert verification: false
[2019-11-12T10:45:41Z DEBUG habitat_http_client::api_client] User-Agent: hab/0.88.0/20191009205151 (x86_64-darwin; 19.0.0)
[2019-11-12T10:45:41Z DEBUG habitat_http_client::api_client] No proxy configured for url: "https://bldr.habitat.sh/v1"
[2019-11-12T10:45:41Z DEBUG habitat_http_client::api_client] Processing cert file: /Users/irvin/.hab/cache/ssl/cert.pem
[2019-11-12T10:45:41Z DEBUG habitat_http_client::api_client] Unable to process cert file: /Users/irvin/.hab/cache/ssl/cert.pem, err=One or more parameters passed to a function were not valid.
[2019-11-12T10:45:41Z DEBUG habitat_api_client::builder] Searching for package mybuildings/api with range 0
[2019-11-12T10:45:41Z DEBUG habitat_http_client::api_client] GET https://bldr.habitat.sh/v1/depot/pkgs/search/mybuildings%2Fapi?range=0&distinct=true with ApiClient { endpoint: "https://bldr.habitat.sh/v1", inner: Client }
[2019-11-12T10:45:41Z DEBUG hyper::client::connect::dns] resolving host="bldr.habitat.sh"
[2019-11-12T10:45:41Z DEBUG hyper::client::connect::http] connecting to 151.101.1.209:443
[2019-11-12T10:45:41Z DEBUG tokio_reactor] adding I/O source: 0
[2019-11-12T10:45:41Z DEBUG tokio_reactor::registration] scheduling Write for: 0
[2019-11-12T10:45:41Z DEBUG hyper::client::connect::http] connected to Some(V4(151.101.1.209:443))
[2019-11-12T10:45:41Z DEBUG tokio_reactor::registration] scheduling Read for: 0
[2019-11-12T10:45:41Z DEBUG tokio_reactor::registration] scheduling Read for: 0
[2019-11-12T10:45:41Z DEBUG tokio_reactor::registration] scheduling Read for: 0
[2019-11-12T10:45:41Z DEBUG tokio_reactor::registration] scheduling Read for: 0
[2019-11-12T10:45:41Z DEBUG tokio_reactor::registration] scheduling Read for: 0
[2019-11-12T10:45:41Z DEBUG tokio_reactor::registration] scheduling Read for: 0
[2019-11-12T10:45:41Z DEBUG hyper::proto::h1::io] flushed 416 bytes
[2019-11-12T10:45:41Z DEBUG hyper::proto::h1::io] read 734 bytes
[2019-11-12T10:45:41Z DEBUG hyper::proto::h1::io] parsed 25 headers
[2019-11-12T10:45:41Z DEBUG hyper::proto::h1::conn] incoming body is content-length (57 bytes)
[2019-11-12T10:45:41Z DEBUG hyper::proto::h1::conn] incoming body completed
[2019-11-12T10:45:41Z DEBUG tokio_reactor] dropping I/O source: 0
[2019-11-12T10:45:41Z DEBUG reqwest::async_impl::response] Response: '200 OK' for https://bldr.habitat.sh/v1/depot/pkgs/search/mybuildings%2Fapi?range=0&distinct=true
[2019-11-12T10:45:41Z DEBUG habitat_api_client::builder] Response Status: 200
No packages found that match 'mybuildings/api'```

**Comment by bixu** [comment]
cc: @arashzandi / @tormen

**Comment by eeyun** [comment]
Closed by #1241
